pkg/common/http.go

```diff
@@ -99,10 +100,13 @@ func PinnedRetryableHttpClient() *http.Client {
 			RootCAs: PinnedCertPool(),
 		},
 		Proxy: http.ProxyFromEnvironment,
-		DialContext: (&net.Dialer{
-			Timeout:   30 * time.Second,
-			KeepAlive: 30 * time.Second,
-		}).DialContext,
+		DialContext: func(ctx context.Context, network, addr string) (net.Conn, error) {
+			d := net.Dialer{
+				Timeout:   30 * time.Second,
+				KeepAlive: 30 * time.Second,
+			}
+			return d.DialContext(ctx, network, addr)
+		},
```

Comment:
Is this functionally the same?

Reply:
it should be. Although I actually don't need to even update this file. The old version had my linter yelling at me so I updated it, but i'll ignore it.